sdk/src/main/java/com/mapbox/maps/CredentialsManager.kt

```diff
@@ -1,51 +0,0 @@
-package com.mapbox.maps
-
-import android.content.Context
-
-/**
- * Convenience class that holds [MapboxMap] related token.
- * `CredentialsManager` could be created per `MapView` instance if given view should use some specific token.
- * Most common use-case when all `MapView`s should use same token could be handled by using `CredentialsManager.default` static object.
- *
- * @property accessToken the token will be applied.
- */
-data class CredentialsManager(private var accessToken: String? = null) {
```

Comment:
Is this deletion aligned wit iOS? AFAIR we added it as part of alignment

Reply:
Not yet, will align with iOS later

Reply:
I guess it will be logical to first align change with iOS and then proceed with merging. We can discuss it today with voice.